geometry_msgs/Pose pose
float32 width
float32 length
float32 score
bool    occupied
